compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/misc/TypedBindingPatternsTest.java

```diff
@@ -116,6 +116,14 @@ public void testErrorBindingPatternInLetVarDecl() {
                 "typed-binding-patterns/typed_binding_patterns_assert_21.json");
     }
 
+    @Test
+    public void testErrorBindingPatternInLocalVarDecl() {
```

Comment:
Since this is an invalid syntax test, we need to add it under the "// Recovery tests" section. Let's add the new test at the very end. Since this is not directly an error binding pattern test, shall we make the test case name, `testErrorBindingPatternInsideListBindingPatternRecovery()`?

Reply:
Updated cdc2ba9492ac40f5768731f83dd248276af7bf08.